templates/.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+    - id: check-yaml
+- repo: git://github.com/antonbabenko/pre-commit-terraform
```

Comment:
maybe consider hosting a `cloudposse/pre-commit` repo with the specific hooks that we want to run? We probably want to control the code (from a security perspective) that's going to execute automatically on each commit.

Also, see this [repo](https://github.com/gruntwork-io/pre-commit) for some useful hooks we might want to use (copy to our own repo).

Reply:
I definitely support creating a cloudposse/pre-commit repo like we with github actions